wagtail/wagtaildocs/views/documents.py

```diff
@@ -34,6 +35,16 @@ def index(request):
         ordering = '-created_at'
     documents = documents.order_by(ordering)
 
+    # Filter by collection
+    current_collection = None
+    collection_id = request.GET.get('collection_id')
+    if collection_id:
+        try:
+            current_collection = Collection.objects.get(id=request.GET['collection_id'])
```

Comment:
Hmm, not sure how that happened :-) Fixed in 174fea121f11fc624ef8a5f2e85b810696d00336, and e62a760d0e4469f82679ca7232d63a5ec92d3691 for images.